### PR TITLE
docs: rehaul website

### PR DESCRIPTION
**Pull request by loredanamoanga**
Check issue #451 

The website is currently staged at : [https://loredanamoanga.github.io/sandbox/](https://loredanamoanga.github.io/sandbox/)
The personal repo will be removed once the website is live

### CODE REVIEW

File: docs/Gemfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
```
Comment (teszes) [inline docs/Gemfile:7]: Do we need this here? What's the point in having a commented out dependency?

File: docs/_config.yml [hunk 2]
```diff
@@ -1,8 +1,30 @@
 ---
+
+# Site settings
 name: Java Operator SDK
-title: Java Operator SDK 
+title: Java Operator SDK
+description:            Build Kubernetes Operators in Java without hassle
+baseurl:                # Site subpath, e.g. /blog
```
Comment (teszes) [inline docs/_config.yml:7]: I'm not familiar with Jekyll, but wouldn't having the subpath either omitted or set to / make sense?

File: docs/_data/navbar.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Navbar menu navigation links
+  - title: Home
+    url: /
+  - title: Docs
+    url: /docs/getting-started
+#     <-- example on how to create navigation dropdown -->
```
Comment (teszes) [inline docs/_data/navbar.yml:6]: We can leave the example here, but is it intentional?

File: docs/_sass/theme/mixins.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+// Mixins
+// ========================================================================
+//.uk-text-muted a {
```
Comment (teszes) [inline docs/_sass/theme/mixins.scss:3]: Is the commented out code here serving a purpose?

File: docs/_sass/theme/uikit.scss [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+@import "../uikit/components/mixin.scss";
+@import "../uikit/components/base.scss";
+
+// Elements
+@import "../uikit/components/link.scss";
+@import "../uikit/components/heading.scss";
+@import "../uikit/components/divider.scss";
+@import "../uikit/components/list.scss";
+@import "../uikit/components/description-list.scss";
+//@import "../uikit/components/table.scss";
```
Comment (teszes) [inline docs/_sass/theme/uikit.scss:14]: Are the commented out lines serving a purpose? Can we either remove them or add a comment explaining why they are here?

File: docs/_sass/theme/variables.scss [hunk 6]
```diff
@@ -0,0 +1,139 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Asap:wght@700&display=swap');
+
+// Global variables
+$global-primary-background:                     #fc9c62;
+//$global-secondary-background:                   #433E3F;
```
Comment (teszes) [inline docs/_sass/theme/variables.scss:6]: Do we need this? Can we remove it instead of commenting it out?

**Review by teszes — CHANGES_REQUESTED** [review]

File: docs/Gemfile (hunk 1 above)
Comment (loredanamoanga) [reply docs/Gemfile]: I was a left over comment, thanks for spotting it!

File: docs/_config.yml (hunk 2 above)
Comment (loredanamoanga) [reply docs/_config.yml]: You are right, I omitted it

File: docs/_data/navbar.yml (hunk 3 above)
Comment (loredanamoanga) [reply docs/_data/navbar.yml]: I moved the example to the docs/readme.md file

File: docs/_sass/theme/mixins.scss (hunk 4 above)
Comment (loredanamoanga) [reply docs/_sass/theme/mixins.scss]: I was a left over comment, thanks for spotting it!

File: docs/_sass/theme/uikit.scss (hunk 5 above)
Comment (loredanamoanga) [reply docs/_sass/theme/uikit.scss]: I added comment explaining why they are commented out, there is an explanation in the readme as well

Comment (loredanamoanga) [reply docs/_sass/theme/uikit.scss]: I also structured them a bit better

File: docs/_sass/theme/variables.scss (hunk 6 above)
Comment (loredanamoanga) [reply docs/_sass/theme/variables.scss]: It was a left over comment, thanks for spotting it!

**Comment by metacosm** [comment]
@loredanamoanga Is there an easy way to check the changes?

**Comment by loredanamoanga** [comment]
> @loredanamoanga Is there an easy way to check the changes?

Yes, you can find it at:  https://loredanamoanga.github.io/sandbox/

**Comment by metacosm** [comment]
How are the releases supposed to be handled?

**Comment by metacosm** [comment]
Could you also add a little information about how would one edit / add stuff on the site, please?

**Comment by metacosm** [comment]
Another thing that is needed, I think, is a `News` page that we could easily edit with a recall of the latest news summary on the home page. Is this something that could be done, @loredanamoanga?

**Comment by loredanamoanga** [comment]
> Could you also add a little information about how would one edit / add stuff on the site, please?

Hi, I have added a readme file inside the docs folder that explains which layouts to use for what and how releases are handled.
To create new pages, or edit them is the regular Jekyll way

**Comment by loredanamoanga** [comment]
> Another thing that is needed, I think, is a `News` page that we could easily edit with a recall of the latest news summary on the home page. Is this something that could be done, @loredanamoanga?

Unfortunately I don't have the time needed for this anymore, but its a good idea!

**Review by metacosm — APPROVED** [review]

**Comment by metacosm** [comment]
@loredanamoanga there's an issue with the site: You are attempting to use the 'none' highlighter, which is currently unsupported on GitHub Pages. Your site will use 'rouge' for highlighting instead. To suppress this warning, change the 'highlighter' value to 'rouge' in your '_config.yml' and ensure the 'pygments' key is unset. For more information, see https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#fixing-highlighting-errors.

**Pull request merged**